roles/test_ha/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#high-availability_assembly-advanced-features
+# Assuming we're in teh right project already...
+
+- name: "Add highAvailability.enabled: true to the spec section"
+  ansible.builtin.shell:
+    cmd: |
+      oc patch stf/default --type merge -p '{"spec":{"highAvailability":{"enabled": true}}}' ; sleep 120
```

Comment:
I think this might not fail correctly if the `oc patch` fails for some reason, because the exit status will get swallowed and we'll get the `sleep` exit status instead. This failure would cause future tests to fail, but the root cause could be obscured. Suggest using `&&` instead of `;` (untested)